install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -168,56 +321,89 @@ To enable writing to NFS volumes with SELinux enforcing on each node, run:
 # setsebool -P virt_use_nfs 1
```

Comment:
There's also a `virt_sandobx_use_nfs` that needs to be enabled.  This is one of the most frequent hangups people hit o this subject.

cc @swagiaal